runtime/cudaq/platform/default/rest/helpers/braket/BraketExecutor.cpp

```diff
@@ -20,6 +20,8 @@
 
 #include <aws/core/utils/ARN.h>
 
+#include <iostream>
+
```

Comment:
If the answer to https://github.com/NVIDIA/cuda-quantum/pull/2416/files#r1859223760 is that it is leftover debug code, then perhaps this new `#include` is no longer necessary?

Reply:
removed, thanks!